app/main.py

```diff
@@ -2,0 +1,27 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "mv" and len(command) == 3:
+        try:
+            mv, file1_name, destination = command
+            if destination.endswith("/"):
+                raise ValueError("Enter correct path and file name!")
+            dirs = destination.split("/")[:-1]
+            file2_name = destination.split("/")[-1]
```

Comment:
The `.split()` method is called twice on the `destination` string. According to the checklist, you should call needed methods only once to improve efficiency .